resolwe_bio/tools/geneset2storage.py

```diff
@@ -19,7 +21,12 @@ def main():
     with open(args.geneset_file) as infile:
         geneset = [gene.strip() for gene in infile]
         data = {'geneset_json': geneset}
```

Comment:
Rename the inner `geneset_json` to `genes` and fix generators, endpoints, queries... Discuss with the product team.